src/directives/center.js

```diff
@@ -185,8 +186,13 @@ angular.module("leaflet-directive").directive('center',
                             map.setView([defaults.center.lat, defaults.center.lng], defaults.center.zoom);
                             leafletEvents.notifyCenterChangedToBounds(leafletScope, map);
                         }
-                    });
+                    };
                 }
+                map.on(events);
```

Comment:
Using an object of events is the cleanest way to keep add bulk events and remove bulk. However, with context of {eventName: "eventName"} we cannot use this everywhere.